ampd/src/event_processor.rs

```diff
@@ -60,7 +60,15 @@ where
             .change_context(Error::EventStream)?;
 
         if let StreamStatus::Active(event) = &stream_status {
-            handler.handle(event).await.change_context(Error::Handler)?;
+            future::with_retry(
+                || handler.handle(event),
+                // TODO: make timeout and max_attempts configurable
+                RetryPolicy::RepeatConstant(Duration::from_secs(1), 3),
+            )
+            .await
+            .change_context(Error::Handler)?;
```

Comment:
Don't we need to add this for the other handlers?

Reply:
Also, I am not sure that we actually want this behavior for multisig handler, because the multisig handler is only talking to tofnd. Maybe we don't want to just ignore in this case?

Reply:
oh wait, disregard, we aren't ignoring, I missed the `?`

Reply:
Ok sorry, I realized I read this wrong initially. So this retry logic is applied to all handlers. But it doesn't like look we are ignoring failures in the case of the evm_verify_msg right? Don't we want to try 3 times, and if we still can't succeed, just continue on? Instead of failing the whole task.

Reply:
which failures are you talking about?

Reply:
any failure really, but specifically rpc error, Wasn't that the requirement? If we try to verify a msg, and encounter an rpc failure, try again. If we try 3 times and still don't succeed, just skip.